src/modules/notes/controllers/deleteNote.js

```diff
@@ -0,0 +1,34 @@
+import Notes from "../models/NoteModel.js";
+
+const ERROR_STATUS = 400;
+const ERROR_NO_MATCHES = "Note not found";
+
+const DeleteNoteController = (request, response, next) => {
+	if (!request.user._id) {
+		const err = new Error("Unauthorized");
```

Comment:
Move to the constants:
`const ERROR_MESSAGES = {}` and `const ERROR_STATUSES = {}`